taichi/transforms/ir_printer.cpp

```diff
@@ -60,7 +60,7 @@ class IRPrinter : public IRVisitor {
   }
 
   void visit(AssertStmt *assert) override {
-    print("{} : assert {}, \"{}\"", assert->id, assert->val->name(),
+    print("{} : assert {}, \"{}\"", assert->id, assert->cond->name(),
           assert->text);
```

Comment:
Maybe we should include the arguments here as well?

Reply:
Done. Now it looks like this:
```
14 : assert $13, "Accessing Tensor of Size [1200, 100] with indices (%d, %d)", $1, $2
```

Reply:
Awesome!!